src/coreComponents/constitutive/solid/PoroElastic.hpp

```diff
@@ -112,7 +112,7 @@ class PoroElastic : public BASE
    * @brief Catalog name
    * @return Static catalog string
    */
-  static string catalogName() { return string( "Poro" ) + BASE::m_catalogNameString; }
+  static string catalogName() { return string( "OldPoro" ) + BASE::m_catalogNameString; }
```

Comment:
This class will be deprecated in a follow on, or as part of this PR?

Reply:
I am going to try to deprecate it in here. Let's see how many tests failures I get....